src/runtimes/docker/clients/DockerClientBase/parseDockerLikeEnvironmentVariables.ts

```diff
@@ -1,26 +1,26 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Git wants to change the line endings. No idea why, but it's going to keep trying to do it, so may as well just check it in.